File: packages/expo-dev-menu/vendored/react-native-gesture-handler/android/devmenu/com/swmansion/gesturehandler/BaseGestureHandlerInteractionController.java
package devmenu.com.swmansion.gesturehandler;

public abstract class BaseGestureHandlerInteractionController
        implements GestureHandlerInteractionController {

  @Override
  public boolean shouldWaitForHandlerFailure(GestureHandler handler,
                                             GestureHandler otherHandler) {
    return false;
  }

  @Override
  public boolean shouldRequireHandlerToWaitForFailure(GestureHandler handler,
                                                      GestureHandler otherHandler) {
    return false;
  }

  @Override
  public boolean shouldRecognizeSimultaneously(GestureHandler handler,
                                               GestureHandler otherHandler) {
    return false;
  }
}

File: packages/expo-dev-menu/vendored/react-native-gesture-handler/android/devmenu/com/swmansion/gesturehandler/BaseGestureHandlerInteractionController.kt
package devmenu.com.swmansion.gesturehandler

abstract class BaseGestureHandlerInteractionController : GestureHandlerInteractionController {
  override fun shouldWaitForHandlerFailure(
    handler: GestureHandler<*>,
    otherHandler: GestureHandler<*>,
  ) = false

  override fun shouldRequireHandlerToWaitForFailure(
    handler: GestureHandler<*>,
    otherHandler: GestureHandler<*>,
  ) = false

  override fun shouldRecognizeSimultaneously(
    handler: GestureHandler<*>,
    otherHandler: GestureHandler<*>,
  ) = false
}
